src/Storage/Storage/Common/InvalidCloudFileShare.cs

```diff
@@ -1,546 +0,0 @@
-﻿// ----------------------------------------------------------------------------------
-//
-// Copyright Microsoft Corporation
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-// http://www.apache.org/licenses/LICENSE-2.0
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-// ---------------------------------------------------------------------------------
-
-namespace Microsoft.WindowsAzure.Commands.Common.Storage.ResourceModel
-{
-    using Microsoft.Azure.Storage;
-    using Microsoft.Azure.Storage.Auth;
-    using Microsoft.Azure.Storage.Core.Util;
-    using Microsoft.Azure.Storage.File;
-    using Microsoft.Azure.Storage.File.Protocol;
-    using Microsoft.Azure.Storage.Shared.Protocol;
-    using System;
-    using System.Collections.Generic;
-    using System.IO;
-    using System.Text;
-    using System.Threading;
-    using System.Threading.Tasks;
-
-    /// <summary>
-    /// This class is used to handle the Azure file which can't be handle in Track1 File Object, like .
-    /// </summary>
-    public class InvalidCloudFileShare : CloudFileShare
```

Comment:
should we also remove "InvalidCloudFileDirectory" and "InvalidCloudFile"?

Reply:
No as they're still being referenced when we need to convert Track2 objects to Track1 objects.